kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/BaseOperation.java

```diff
@@ -540,7 +540,7 @@ public String getLabelQueryParam() {
           sb.append(",");
         }
         Map.Entry<String, String[]> entry = iter.next();
-        if (entry.getValue() != null) {
+        if (entry.getValue() != null && Utils.isNotNull(entry.getValue())) {
```

Comment:
First null check is redundant, `Utils#isNotNull`already handles that case
```suggestion
        if (Utils.isNotNull(entry.getValue())) {
```